Source/WebCore/page/ScriptTelemetryCategory.cpp

```diff
@@ -62,6 +62,8 @@ ASCIILiteral description(ScriptTelemetryCategory category)
         return "ScreenOrViewport"_s;
     case ScriptTelemetryCategory::Speech:
         return "Speech"_s;
+    case ScriptTelemetryCategory::Elements:
```

Comment:
Maybe this could be more specific (`FormControls` or `InputValue`?)

Reply:
Yeah, I'll change this to `FormControls`